oauth2client/django_orm.py

```diff
@@ -46,12 +46,16 @@ def to_python(self, value):
             return None
         if isinstance(value, oauth2client.client.Credentials):
             return value
-        return pickle.loads(base64.b64decode(value))
+        return pickle.loads(base64.b64decode(smart_bytes(value)))
 
-    def get_db_prep_value(self, value, connection, prepared=False):
```

Comment:
@dhermes sure:
* the relationship between `get_db_prep_value` and `get_prep_value`:
https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.Field.get_db_prep_value
* the relationship between `value_to_string` and serialization: https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/#converting-field-data-for-serialization